src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [
+    {
+      id: 1,
+      title: '48 Laws of Power',
+      author: 'Robert Greene',
+      category: 'Personal Development',
+    },
+    {
+      id: 2,
+      title: 'Personal Development',
+      author: 'Jim Rohn',
+      category: 'Personal Development',
+    },
+  ],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
+  },
+});
```

Comment:
- [ ] The previous reviewer forgot to mention this as well but as part of the project requirements please make sure to check the action payload before returning the message as shown below, this means you'll have to add parameters `state` and `action` as this is a reducer
```jsx
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },

```